src/main/java/sggw/wzim/czasnawypad/controller/AttractionFavouriteController.java

```diff
@@ -32,5 +32,10 @@ ResponseEntity<?> addFavourite(@RequestBody @Valid CreateFavouriteAttractionDTO
         return ResponseEntity.ok().body(result);
     }
 
+    @DeleteMapping("/{id}")
+    ResponseEntity<?> deleteFavourite(@PathVariable Integer id) {
+        List<FavouriteAttractionDTO> deleteInfo = favouriteService.deleteFavourite(id);
+        return ResponseEntity.ok(deleteInfo);

```

Comment:
We don't return anything when deleting resource. Drop this implementation and provide one that returns just 204 No Content